network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -292,26 +293,44 @@ class ProductPageEvaluation(models.Model):
 
     @property
     def total_votes(self):
-        return self.votes.count()
```

Comment:
**suggestion:** Add a comment indicating which QuerySet method should be used to populate `_total_votes`